src/olympia/stats/tests/test_views.py

```diff
@@ -912,3 +912,19 @@ def test_skips_none_key(self):
         series = views.process_locales(series)
 
         assert len(next(series)['data']) == 0
+
+
+class TestRenderCSV(TestCase):
+    def test_handles_null_keys(self):
+        series = [
+            {'data': {None: 1, 'a': 2}, 'count': 3, 'date': '2020-01-01'},
+            {'data': {'a': 4}, 'count': 4, 'date': '2020-01-02'},
+        ]
+
+        stats, fields = views.csv_fields(series)
+        response = views.render_csv(request=RequestFactory().get('/'),
+                                    addon=addon_factory(),
+                                    stats=stats,
+                                    fields=fields)
+
+        assert '\n,a\r\n1,2\r\n0,4\r\n' in force_text(response.content)
```

Comment:
It'd be:
```
,a
1,2
0,4
```
?

Reply:
Yes. I guess I can use a `'\r\n'.join([',a', '1,2', '0,4'])` instead if the comment is about readability. Otherwise, I think using a multiline string won't work because of the line endings.

Reply:
I think that's clearer, yeah.  (I wrote it out because that was the only way I could parse it - doing it in my head didn't work!)